doc/guides/SimpleStartupGuide.md

```diff
@@ -305,6 +309,25 @@ Follow these instructions to update devices.
    ```bash
    [{"apiVersion":"v2","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
    ```
+</details>
+
+<details>
+
+<summary><strong>Auto-Discovery</strong></summary>  
+
+<br/>
+
+ONVIF devices support WS-Discovery, which is a mechanism that supports probing a network to find ONVIF capable devices.  Refer to [How does WS-Discovery work?](https://github.com/EdgeX-Camera-Management/device-onvif-camera/blob/main/doc/ws-discovery.md) and [Auto Discovery](https://github.com/EdgeX-Camera-Management/device-onvif-camera/blob/main/doc/auto-discovery.md) for more information auto-discovery mechanism.
+
+> **NOTE:** Ensure that the cameras are all installed and configured before attempting discovery.
+
+
+1. Set the DiscoverySubnets by running `bin/configure-subnets.sh`.
+
+2. Device discovery is triggered by the device SDK. Once the device service starts, it will discover the Onvif camera(s) at the specified interval.
+> **Note:** You can also manually trigger discovery using this command: `curl -X POST http://<service-host>:59984/api/v2/discovery`
+
+</details>
 
 1. Map credentials using the `map-credentials.sh` script.  
    a. Run `bin/map-credentials.sh`    
```

Comment:
@ChristianDarr-personal
https://github.com/edgexfoundry/device-onvif-camera/pull/161#discussion_r973302960
There appears to be some inconsistency throughout the doc with regards to colons or periods at the end of numbered instructions. I'm not sure but it looks like we may be using a colon if the following line is a command that needs to be entered and a period if its just another sentence. Please let me know if I should replace all the periods.